README.md

```diff
@@ -108,7 +108,16 @@ Update your project configuration to use Cmark:
 docs: [markdown_processor: ExDoc.Markdown.Cmark]
 ```
 
-# License
+## Contributing
+
+The easiest way to test changes to ExDoc is to locally re-generate it's own docs:
+
+1. Run `mix setup` to install all dependencies
```

Comment:
`mix setup` uses just npm, however in https://github.com/elixir-lang/ex_doc/blob/master/assets/README.md#dependencies we mention yarn. I think we can assume folks installed yarn, so mix setup calls `yarn install` etc; wdyt?

Reply:
Agreed. We should use yarn. But I think after the first install they would behave the same.